spring-boot-admin-server-ui/src/main/frontend/views/applications/ApplicationStatusHero.vue

```diff
@@ -19,6 +19,16 @@
             <p class="text-gray-400" v-text="lastUpdate" />
           </div>
         </template>
+        <template v-else-if="notUpCount > 0">
```

Comment:
this means that the "unknown" message is also shown for "OFFLINE", "RESTRICTED", "OUT_OF_SERVICE" ...? Is this intended? Or should we interpret some of them as "down"?

Reply:
You are absolutely right. I will rework this part and add some more advanced state messages.